docs/building/windows.md

```diff
@@ -4,69 +4,46 @@ This document is written with the intention of demonstrating the build process i
 
 **Note**: This guide was written for Windows 10, however the process should be the same for Windows 11. Please make an issue if you come across a Windows 11 specific issue and it will be addressed.
 
-## Getting Started / Prerequisites  
-
-The following software is required:
-
-```
-git
-nodejs
-```
-
-### Instal git
-
-**note**: If you do not plan to contribute to the project, git is not required as the repo can downloaded [directly](https://github.com/alephium/desktop-wallet/archive/refs/heads/master.zip)
-
-After setting up an account on Github, download [Github Desktop](https://desktop.github.com/).
-
+## Getting Started / Prerequisites
 
 ### Instal NodeJS
 
-Navigate to the [NodeJS Offical Installer](https://nodejs.org/en/download/) and download the Windows installer proper for your architecture (for this guide we will elect the `x64` in `.msi` format)
+Navigate to the [NodeJS Offical Installer](https://nodejs.org/en/download/) and download the Windows installer proper for your architecture (the most common choice is `x64` in `.msi` format)
 
 After the download completes, open the installer and complete the wizard (all defaults are fine)
 
-### (Optional) Install a Text Editor
-
-A text editor like [VSCode](https://code.visualstudio.com/docs/?dv=win) is recommended if you plan to contribute or work on the project.
-
-
-
-
 
 ## Building
 
 
 ### Clone the Project
 
-Begin by cloning or downloading the `desktop-wallet` [repo](https://github.com/alephium/desktop-wallet). This can be accomplished by selecting the green `<>Code` button and then electing either `Open with Github Desktop` or `Download ZIP`.
+Download the `desktop-wallet` [repo](https://github.com/alephium/desktop-wallet).
 
-**Note**: If you plan to make changes, its a good idea to go ahead and [create a new branch](https://www.softwaretestinghelp.com/github-desktop-tutorial/) before starting.
+This can be accomplished by selecting the green `<>Code` button and then `Download ZIP` from the dropdown.
+
+Or by following this [link](https://github.com/alephium/desktop-wallet/archive/refs/heads/master.zip).
```

Comment:
Target latest release, not master

Reply:
got it! Good catch!

Reply:
There is no `release` branch so if I target a specific release the documentation will need updated (or will be out of date) every release cycle.

Reply:
For now use https://github.com/alephium/desktop-wallet/releases/tag/latest even if it's not present. I'll deal with this tomorrow

Reply:
/cc @mvaivre @nop33

Reply:
got it!

Reply:
You use this link which always points to the latest release: https://github.com/alephium/desktop-wallet/releases/latest
Using `master` is not a good idea, as the code is not always made to work with mainnet, but with a future version of node/explorer.

Reply:
but we will have to keep your tag always up to date when we release a new release. The link I shared always points to the latest release without any additional work.